packages/shared/src/domain/dob.ts

```diff
@@ -1,14 +1,15 @@
-import { buildDayjs, ISO_DATE, validateDateOfBirth } from "../common/date";
+import { buildDayjs, ISO_DATE, validateDateOfBirthSafe } from "../common/date";
+import { BadRequestError } from "../error/bad-request";
 
 export function normalizeDobSafe(date: string): string | undefined {
   const trimmedDate = date.trim();
   if (trimmedDate.length < 1) return undefined;
-  if (!validateDateOfBirth(trimmedDate)) throw new Error("Invalid date of birth.");
+  if (!validateDateOfBirthSafe(trimmedDate)) return undefined;
   return buildDayjs(trimmedDate).format(ISO_DATE);
 }
```

Comment:
lol we were throwing an error inside of something called normalizeDobSafe?? 😬

Reply:
Worse, we only call it on a function that throws an error if the return is undefined... 😆